docs/docs/policies/index.md

```diff
@@ -43,6 +43,6 @@ This policy applies to any workload (global) and will log the dropped_executable
 While specifying event filters is optional, policies must have the `name`, `description`, `scope` and `rules` fields.
 
 !!! Note
-    A current limitation is that only one rule can be defined per any event type in a policy
+    Note that only one rule can be defined per any event type in a policy
```

Comment:
I don't see why this is a limitation -- I know users should be aware of it but I would not call it a limitation

Reply:
Ok, but I would add the word "currently" since we want to support more than one rule per event in the future

Reply:
oki, sounds good, will add it